change/@fluentui-web-components-057bb4b4-9195-499c-a1c4-4fdb458f1302.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
I don't think you're updating the web components package, so this should be a `none` change type.